apps/condo/domains/property/components/panels/Builder/MapConstructor.ts

```diff
@@ -382,6 +388,27 @@ class MapEdit extends MapView {
         this.mode = mode
     }
 
+    public formRef: React.MutableRefObject<HTMLDivElement | null> 
+
+    public setRefToForm (ref: React.MutableRefObject<HTMLDivElement | null>): void {
+        this.formRef = ref
+    }
+
+    private scrollToForm (): void {
```

Comment:
Now builder has technology specific context, try to avoid this synthetic connection. Guess, it's better to add callback and undepended handler which will isolated from concreete realization of tehcnology.

Reply:
Done